config/base/crds/kafka.banzaicloud.io_kafkatopics.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
not part of this patch right?

Reply:
I noticed upstream was updated to this version, had the same curiosity.
Version comes from configured kubebuilder CONTROLLER_GEN_VERSION : https://github.com/adobe/koperator/blob/6396af5b5751e88637599e6c7e8c97b39d41cf87/Makefile#L20

It seems that this was a change that probably escaped the initial dependencies update.